### PR TITLE
Switch develop

### PR DESCRIPTION
**Pull request by yebata**
To implement ISX-255, at first added device orientation as a usage to a device.
I improved it to be able to add/remove multiple usages in commit d608040, but it seemed to be in an another PR so I backed out it once.

### CODE REVIEW

**Comment by jechter** [comment]
Please add a CHANGELOG entry

**Review by douglas-piconi — APPROVED** [review]
Tests ran on staging branch.  Green CI. 

https://github.com/Unity-Technologies/InputSystem/commits/switch-ci-run

**Comment by yebata** [comment]
Added the change in the Add section in the change log

**Review by jechter — APPROVED** [review]

**Comment by douglas-piconi** [comment]
Latest commit pass CI run     green.  https://github.com/Unity-Technologies/InputSystem/commits/switch-ci-run

**Comment by Rene-Damm** [comment]
Dropped the ball on this one. Will pull this in through a separate PR once #734 with the ability to have multiple usages has landed in the develop branch.

File: Packages/com.unity.inputsystem/InputSystem/Plugins/Switch/NPad.cs [hunk 1]
```diff
@@ -484,17 +484,46 @@ protected override void RefreshConfiguration()
...
+                    InputSystem.AddDeviceUsage(this, CommonUsages.LeftHand);
+                }
+                else if (m_StyleMask == NpadStyle.JoyRight)
+                {
+                    InputSystem.AddDeviceUsage(this, CommonUsages.RightHand);
+                }
+                else if (m_StyleMask == NpadStyle.JoyLeft)
+                {
+                    InputSystem.AddDeviceUsage(this, CommonUsages.LeftHand);
+                }
```
Comment (Rene-Damm) [inline Packages/com.unity.inputsystem/InputSystem/Plugins/Switch/NPad.cs:513]: Looking at this again, the code here shouldn't use `SetDeviceUsage` but should instead rely entirely on `AddDeviceUsage` and `RemoveDeviceUsage` in order to preserve any usages the user may have set.

Comment (yebata) [reply Packages/com.unity.inputsystem/InputSystem/Plugins/Switch/NPad.cs:513]: Thanks, Rene. Martin noticed me that these changes should be in an NDA repo, so I'll fix it as you pointed out in there.

Comment (Rene-Damm) [reply Packages/com.unity.inputsystem/InputSystem/Plugins/Switch/NPad.cs:513]: Thank you @yebata. I'm sorry the changes didn't make it before the split-off. I kept having it on my list but then didn't get it done before Switch was moved out.

**Pull request closed without merging**